### DISCUSSION TITLE
installing chaos agents in different clusters

### DISCUSSION THREAD

**Discussion by vinaymathew** [litmuschaos/litmus#3375] [Q&A] [answered]
Hi Everyone,
My team is starting out new in this domain and are impressed by what Litmuschaos offers us out of the box. We would like to use this in our work but we have a constellation where we have multiple K8s clusters from different teams in different AWS accounts for example. We have a central place like a AWS management account where we host centralized applications and litmuschaos would be one such. What we would like to do is to be able to have a centralized ChaosCenter where different teams have their own accounts. Chaos agents installed on every teams K8s cluster should be able to be connected to the main chaos center.. the mother ship if you will. I have been trying to figure out whether its possible to install chaos agents on remote clusters using litmusctl and attach them to the main chaos center but was not very successful on this. So just wanted to ask if a set up like this is possible with LitmusChaos today?
Thanks,
Vinay

**Comment by imrajdas** [comment, marked as answer]
Hi, You can connect an infinite number of chaos agents to different clusters. 

Please follow this doc- https://docs.litmuschaos.io/docs/litmusctl/usage-interactive-mode
